src/main/java/app/controller/LabelController.java

```diff
@@ -26,11 +26,12 @@
 public class LabelController {
     private final LabelService labelService;
 
-    @PostMapping
+    @PostMapping("/{taskId}")
```

Comment:
It is bad practice to add ID of another entity here. It is best to use RequestParam anotation before taskId and remove this.